service/history/transferQueueActiveTaskExecutor_test.go

```diff
@@ -2499,6 +2499,7 @@ func (s *transferQueueActiveTaskExecutorSuite) createAddActivityTaskRequest(
 		ScheduledEventId:       task.ScheduledEventID,
 		ScheduleToStartTimeout: ai.ScheduleToStartTimeout,
 		Clock:                  vclock.NewVectorClock(s.mockClusterMetadata.GetClusterID(), s.mockShard.GetShardID(), task.TaskID),
+		WorkerVersionStamp:     nil, // FIXME: test with something here?
```

Comment:
I'm guessing this is meant to be addressed in the follow up on activity tasks, but just checking you didn't mean to deal with in in this PR.